libs/SmartSync/SmartSyncTests/SyncManagerTests.m

```diff
@@ -1074,7 +1123,7 @@ - (void)testCustomSyncUpWithFetchModificationDateFailure
     SFSyncUpTarget *customTarget = [[TestSyncUpTarget alloc] initWithRemoteModDateCompare:TestSyncUpTargetRemoteModDateGreaterThanLocal
                                                                                      sendRemoteModError:YES
                                                                                         sendSyncUpError:NO];
-    [self trySyncUp:3 actualChanges:1 target:customTarget mergeMode:SFSyncStateMergeModeLeaveIfChanged completionStatus:SFSyncStateStatusFailed];
+    [self trySyncUp:3 actualChanges:3 target:customTarget mergeMode:SFSyncStateMergeModeLeaveIfChanged completionStatus:SFSyncStateStatusDone];
```

Comment:
The behavior was different from android before. Now it won't fail the sync up if the server date cannot be retrieved.